contracts/metatx/BaseRelayRecipient.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../utils/Context.sol";
+
+/*
+ * @dev Context variant with ERC2771 support.
+ */
+abstract contract BaseRelayRecipient is Context {
```

Comment:
We want to change this name. Some candidates are:

- `MetaTxContext`
- `ForwarderContext`
- `ForwardedContext`
- `ERC2771Context`
- `RelayedContext`

Reply:
I don't like `ForwarderContext` (might confuse people in thinking this is part of the forwarder)
No strong opinions for the others